pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -739,7 +739,10 @@ func handleConnectionRequestEvents(t *testing.T, svc *Service, e dispatcher.DIDC
 	}(e)
 
 	// invoke callback
-	e.Callback(dispatcher.DIDCommCallback{Err: err})
+	msg := &dispatcher.DIDCommCallback{}
+	err = msg.Done(dispatcher.Result{Err: err})
+	require.NoError(t, err)
+	e.Callback(msg)
```

Comment:
Can you please add a test case to validate the error from callback processing ? i.e, consumer calls the callback and validates the error from the callback.

Reply:
Done, I've modified the existing one